packages/react-storage/src/components/StorageManager/index.ts

```diff
@@ -1,2 +1,9 @@
 export { StorageManager } from './StorageManager';
 export { StorageManagerProps } from './types';
+export {
+  DropZoneProps,
```

Comment:
Exporting these prop types to make the component overrides easier for customers

Reply:
Think you will need to add them to _src/components/index.ts_ and _src/index.ts_

Reply:
YOU will need to export them

Reply:
FINE

Reply:
done